recipes-connectivity/networkmanager/networkmanager-1.4.4/dbus-system.conf

```diff
@@ -0,0 +1,21 @@
+<!--
+This configuration file is no longer required and may be removed.
+
+In older versions of dbus, this file defined the behaviour of the well-known
+system bus. That behaviour is now determined by
+/usr/share/dbus-1/system.conf, which should not be edited.
+
+For local configuration changes, create a file
+system-local.conf or files matching system.d/*.conf in the same directory
+as this one, with a <busconfig> element containing configuration directives.
+These directives can override D-Bus or OS defaults.
+
+For upstream or distribution-wide defaults that can be overridden
+by a local sysadmin, create files matching
+/usr/share/dbus-1/system.d/*.conf instead.
+-->
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-Bus Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+<allow_anonymous/>
```

Comment:
Yes, I will drop the above comment and put a rationale.

As I understand DBUS_COOKIE_SHA1, the cookie stored is updated periodically and nm-applet is required to prove access to the cookie file by reading the cookie. I'm not sure this can be disabled to have one static cookie. Also glib (nm-applet) should be patched to read cookie from domstore or otherwise. This could be researched more and can be taken up as a separate task.

Reply:
This is beyond the scope of this work, but it would be good if we expanded the dbus auth cookie code to be able to put it in a location accessible by domains that would like to use dbus. Something to consider when suggesting domstore is that domstore is really a dbus service and thus you are publishing the auth cookie, who's premise is out-of-band exchange, over dbus.

Reply:
Oh, I didn't realize `DBUS_COOKIE_SHA1` changed. Yes, that doesn't make it a good fit.